apps-rendering/src/renderer.ts

```diff
@@ -177,12 +177,13 @@ const plainTextElement = (node: Node, key: number): ReactNode => {
 const shouldShowDropCap = (
 	text: string,
 	format: ArticleFormat,
+	isFirstParagraph: boolean,
 	isEditions: boolean,
 ): boolean => {
```

Comment:
Tiny non-blocking suggestion if you think its worth it:

Could these parameters be made an object so when reading invocations of this functions its easy to see what the parameters are rather than just 'true' or 'false'. `isEditions` might benefit from a default value of `false` so it doesn't need to be passed unless required.

```suggestion
const shouldShowDropCap = (
{
	text,
	format,
	isFirstParagraph,
	isEditions,
}): boolean => {
```

Reply:
I agree, the only reason I won't refactor further is because I hope all this code will be redundant once DCAR is complete in January. And then we can delete it all. If not, someone can read this comment in 5 years time and be very amused.

Reply:
Makes sense! 👍